edb/server/bootstrap.py

```diff
@@ -2484,13 +2491,13 @@ async def _bootstrap(ctx: BootstrapContext) -> edbcompiler.CompilerState:
     if backend_params.has_create_database:
         await _create_edgedb_database(
             ctx,
-            edbdef.EDGEDB_SUPERUSER_DB,
-            edbdef.EDGEDB_SUPERUSER,
+            default_branch,
+            args.default_database_user or edbdef.EDGEDB_SUPERUSER,
         )
     else:
         await _set_edgedb_database_metadata(
             ctx,
-            edbdef.EDGEDB_SUPERUSER_DB,
+            default_branch,
```

Comment:
We should also use `default_branch` in `system_api.py` if not `has_create_database`?

> ```
> async def _ping(tenant):
>     if tenant.get_backend_runtime_params().has_create_database:
>         dbname = edbdef.EDGEDB_SYSTEM_DB
>     else:
>         dbname = edbdef.EDGEDB_SUPERUSER_DB  # FIXME
> ```